lib/dotcom_web/components/trip_planner/itinerary_summary.ex

```diff
@@ -99,11 +109,17 @@ defmodule DotcomWeb.Components.TripPlanner.ItinerarySummary do
       <%= for {route, index} <- Enum.with_index(@routes) do %>
         <.route_symbol route={route} class={"#{@grouped_classes} #{zindex(index)} #{@class}"} />
 
-        <div
-          :if={@slashed? and index < Kernel.length(@routes) - 1}
-          class={"bg-white -mt-0.5 w-1 h-7 #{zindex(index)} transform rotate-[17deg]"}
-        >
-        </div>
+        <%= index < Kernel.length(@routes) - 1 do %>
+          <%= if if @slashed? do %>
+            <div
+              class={"bg-white -mt-0.5 w-1 h-7 #{zindex(index)} transform rotate-[17deg]"}
+              aria-label="or"
+              role="text"
```

Comment:
I don't believe "text" is a [valid ARIA role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles), you can just remove it